pkgs/development/python-modules/blist/default.nix

```
@@ -14,6 +15,13 @@ buildPythonPackage rec {
     sha256 = "1hqz9pqbwx0czvq9bjdqjqh5bwfksva1is0anfazig81n18c84is";
   };
 
+  patches = [
+    (fetchpatch {
```

can you add a comment stating what the patch is for?
```suggestion
    # python3.7 compatibility
    (fetchpatch {
```

---

resolved